jormungandr/src/settings/command_arguments.rs

```diff
@@ -61,6 +61,11 @@ pub struct StartArguments {
     /// or from the network.
     #[structopt(long = "genesis-block-hash", parse(try_from_str))]
     pub block_0_hash: Option<HeaderHash>,
+
+    // TODO: Explorer mode description
+    // Maybe a config file variable is better
```

Comment:
A config file boolean is good too. But the command line option still have the priority over the config file (i.e. if someone set `--enable-explorer=false` then we don't allow the explorer module and handlers in the REST interface).